src/decoder/mp3.rs

```diff
@@ -14,8 +14,9 @@ where
 {
     // decoder: SeekDecoder<R>,
     decoder: Decoder<R>,
-    current_frame: Frame,
-    current_frame_offset: usize,
+    // minimp3 Frames are interpreted as rodio spans
```

Comment:
Maybe: `// What minimp3 calls frames rodio calls spans`

and then fully qualify the path to `Frame` (something like `minimp3::Frame`)?